modules/nw-openshift-sdn-modes.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/about-openshift-sdn.adoc
+
+[id="nw-openshift-sdn-modes_{context}"]
+= OpenShift SDN network isolation modes
+
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+OpenShift SDN provides three SDN modes for configuring the pod network:
+
+* The _network policy_ mode allows project administrators to configure their own
+isolation policies using NetworkPolicy objects. Network policy is the default mode in {product-title} {product-version}.
```

Comment:
Shouldn't `NetworkPolicy` here be in backticks?

Reply:
@bobfuru, yes! That somehow was missed when updating styles sometime ago.